.ado/pipelines/config/variables-values-e2e.yaml

```diff
@@ -9,7 +9,7 @@ variables:
 # The first value in 'stampLocations' is the primary region used for global services.
 # IMPORTANT! Changing the primary region (first value) is a BREAKING change and will destroy CosmosDB and Front Door.
 - name: 'stampLocations'
-  value: '["swedencentral"]'    # Check which regions are valid. There is a list in /src/infra/README.md
+  value: '["uksouth"]'    # Check which regions are valid. There is a list in /src/infra/README.md
```

Comment:
will be reverted. But we cannot use sweden to run chaos experiments. Only selected regions work as targets during the preview

Reply:
I had to remove 'swedencentral' as a secondary region. The current pipeline logic is applying chaos to the last (in this case second) stamp and that was 'swedencentral' where chaos is not yet available.

Reply:
actually, it *should* apply it to the first region... just turns out that the terraform output is not ordered, so sometimes it ends up the other regions, I guess. I'll look into it. For the demo we need two regions.

    $stamp = $releaseUnitInfraDeployOutput.stamp_properties.value[0]

Reply:
fixed with enhanced logic